### PR TITLE
Bring 1.14 fees in line with 1.10

### PR DESCRIPTION
**Pull request by rnicoll**
Reduce fee defaults to saner amounts

### CODE REVIEW

File: src/wallet/wallet.h [hunk 1]
```diff
@@ -47,9 +47,9 @@ static const unsigned int DEFAULT_KEYPOOL_SIZE = 100;
 //! -paytxfee default
 static const CAmount DEFAULT_TRANSACTION_FEE = 0;
 //! -fallbackfee default
-static const CAmount DEFAULT_FALLBACK_FEE = COIN * 20;
+static const CAmount DEFAULT_FALLBACK_FEE = COIN;
 //! -mintxfee default
-static const CAmount DEFAULT_TRANSACTION_MINFEE = COIN;
+static const CAmount DEFAULT_TRANSACTION_MINFEE = COIN / 10;
```
Comment (langerhans) [inline src/wallet/wallet.h]: Why ist this `COIN / 10`? From 1.10: `CFeeRate minRelayTxFee = CFeeRate(COIN);
`

Comment (rnicoll) [reply src/wallet/wallet.h]: The idea is to intentionally loosen the minimum so we've got some scope for the price to continue upwards without causing problems.

Comment (langerhans) [reply src/wallet/wallet.h]: Given we are always creating transactions at `minRelayTxFee` this change would be effectively reducing the fee for every Core user to 0.1 DOGE. I think you meant to change `DEFAULT_MIN_RELAY_TX_FEE`?

**Comment by Turupawn** [comment]
I don't see a reason to reduce the fee. 1Ð fee is easy to understand and explain to my fellow friends new to the cryptos.
I don't think using decimals (0.1Ð) is making doge easier to understand.
I understand that the dogecoin it's rasing it's value compared to usd.
But last time I checked 1Ð = 1Ð

**Comment by rnicoll** [comment]
So the default amounts will stay the same, so 1 DOGE will be the normal fee, this is just giving us scope for moving to a more flexible model later. Realistically we have to avoid turning into something that's too valuable to use, it would be crazy.

**Comment by GaryLittlemore** [comment]
At the current price $1 is 0.01475808492078891 doge, keeping the fee low is a must. Could it not be coded that the fee would be variable depending on the price but it would always be  equivalent to say 50 Cent?

**Comment by langerhans** [comment]
No, that is not possible as this would require the client knowing about exchange rates. Ain't gonna happen ;) Fees can already be set by users and miners. These are merely default fees. Yet, I'd be fine with keeping them at 1 DOGE as this is just over a penny. Even a tenfold increase in price would make it 10-15 cents which is still a lot better than paying a 50$ fee on a 25$ transactions cause you have some small outputs to spend 😂

**Comment by PeterOdeus** [comment]
The 1-doge fee is a cognitive cornerstone for simplicity. If we're
anticipating a price-increase as a general tidal effect of all crypto, I'd
rather contemplate an event like "The Doubling": A 2:1 increase of both
mining fee and wallet holdings. Effectively halving the Dogecoin price
while maintaining the marketcap.

**Comment by rnicoll** [comment]
I've just realised why everyone's commenting on this...

Shibes, I screwed up the first version of 1.14 and it had 10x regular fees. This was just me putting it back down to 1 DOGE. Stop panicking!

**Pull request merged**